config/eslint/eslintrc.js

```diff
@@ -0,0 +1,205 @@
+module.exports = {
+  env: {
+    browser: true,
```

Comment:
```suggestion
    browser: false,
```

I tried this locally and the linter passed. Not sure why it was `true`.